web/package.json

```diff
@@ -23,6 +23,7 @@
     "antd": "5.2.3",
     "antd-token-previewer": "^1.1.0-22",
     "buffer": "^6.0.3",
+    "caniuse-lite": "^1.0.30001621",
```

Comment:
is this useful?

Reply:
Maybe not. I seem to have added it when executing'go mod tidy 'or other commands. I'll pay attention next time.

Reply:
@Bsheepcoder not next time. It needs to be fixed "this time", or this PR will not pass